docs/api.md

```diff
@@ -612,6 +615,11 @@ const page = await context.newPage();
 await page.goto('https://example.com');
 ```
 
+#### browser.defaultBrowserContext()
```

Comment:
Can we call this the mainBroswerContext to align with mainFrame?

Reply:
There's no hierarchy between browser contexts.